aspsp-mock-server/src/main/java/META-INF/.gitignore

```diff
@@ -0,0 +1 @@
+/MANIFEST.MF
```

Comment:
Why MANIFEST.MF is ignored here? We don't keep it in repo, it is generated by spring boot during the build time and comes only into target folder, not source one